x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/support/ApiKeyBoolQueryBuilder.java

```diff
@@ -110,7 +92,9 @@ protected QueryBuilder doRewrite(QueryRewriteContext queryRewriteContext) throws
     }
 
     static boolean isIndexFieldNameAllowed(String fieldName) {
```

Comment:
This turned out really nice. A small nit is to put the other allowed index fields in a constant static Set in the class.

Reply:
I pushed [d087779](https://github.com/elastic/elasticsearch/pull/109559/commits/d087779c94baae0b1a1f6e50dea561827e7075ab).